airflow-core/src/airflow/configuration.py

```diff
@@ -356,6 +356,7 @@ def sensitive_config_values(self) -> set[tuple[str, str]]:
         ("api", "access_logfile"): ("webserver", "access_logfile", "3.0"),
         ("triggerer", "capacity"): ("triggerer", "default_capacity", "3.0"),
         ("api", "expose_config"): ("webserver", "expose_config", "3.0.1"),
+        ("fab", "expose_hostname"): ("webserver", "expose_hostname", "3.0.1"),
```

Comment:
3.0.1 or 3.0.2?
The 3.0.1 is already in rc

Reply:
Or 3.1.0?